client/src/context/useFindSitterFormContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { FindSitterForm } from '../interface/FindSitterForm';
+import { createContext, FunctionComponent, useState, useContext, useCallback } from 'react';
+
+interface IFindSitterFormContext {
+  findSitterFormContext: FindSitterForm | null | undefined;
+  updateFindSitterFormContext: (findSitterForm: FindSitterForm) => void;
+}
+
+export const FindSitterFormContext = createContext<IFindSitterFormContext>({
+  findSitterFormContext: undefined,
+  updateFindSitterFormContext: () => null,
+});
+
+export const FindSitterFormProvider: FunctionComponent = ({ children }): JSX.Element => {
```

Comment:
Good thinking but we don't need a separate context, you can include the `findSitterFormContext` state within the SittersProvider (the context which contains sitter profiles) itself 😄